src/settings/Locale.js

```diff
@@ -93,7 +95,24 @@ class Locale extends React.Component {
     return JSON.stringify({ locale, timezone, currency });
   }
 
+  handleLocaleChange = (event, newValue) => {
+    const region = newValue.replace('-', '_');
+    fetch(translations[region] ? translations[region] : translations.en_US)
+      .then((response) => {
+        if (response.ok) {
+          response.json().then((values) => {
+            let warning = values['ui-tenant-settings.settings.locale.localeWarning'];
+            const buttonValue = this.props.intl.formatMessage({ id: 'ui-tenant-settings.settings.locale.changeSessionLocale' });
+            warning = warning.replace('{label}', buttonValue);
+            // eslint-disable-next-line no-alert
+            window.alert(warning);
```

Comment:
I mean....sure....no good deed goes unpunished....but don't we have [`ConfirmationModal`](https://ux.folio.org/storybook/?path=/story/modal--confirmationmodal) for stuff like this rather than an `alert()`?